lib/listener/helpers.js

```diff
@@ -11,9 +11,16 @@ module.exports = function () {
   const helpers = container.helpers();
 
   const runHelpersHook = (hook, param) => {
-    if (store.dryRun) return;
+    if (store.dryRun) 
+      return;
+    
     Object.keys(helpers).forEach((key) => {
-      helpers[key][hook](param);
+      if (!helpers[key][hook]) {
+			  console.log(`${key}'s hook value is: ${helpers[key][hook]}`);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Mixed spaces and tabs.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4932579)